hack/validate-deployer-pipeline.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
Are these TWO scripts being used?

Reply:
I just used them, does that count?

Reply:
No, but I don't care :D It's your zombie code :trollface: